corehq/apps/app_manager/models.py

```diff
@@ -633,6 +633,7 @@ class FormBase(DocumentSchema):
     auto_gps_capture = BooleanProperty(default=False)
     no_vellum = BooleanProperty(default=False)
     form_links = SchemaListProperty(FormLink)
+    comment = DictProperty(unicode)
```

Comment:
Seems like you could use a mixin to avoid having to duplicate this field and the related functions.